package.json

```diff
@@ -156,13 +156,13 @@
     "ember-cli-typescript": "^4.1.0",
     "ember-cli-typescript-blueprints": "^3.0.0",
     "ember-code-snippet": "^2.4.0",
-    "ember-collapsible-panel": "^3.1.1",
+    "ember-collapsible-panel": "^6.0.1",
     "ember-component-attributes": "^0.1.1",
     "ember-composable-helpers": "^4.4.1",
     "ember-concurrency": "^2.0.3",
     "ember-concurrency-async": "^1.0.0",
     "ember-concurrency-ts": "^0.2.2",
-    "ember-content-placeholders": "https://github.com/cos-forks/ember-content-placeholders#master#2018-07-19",
+    "ember-content-placeholders": "https://github.com/brianjgeiger/ember-content-placeholders#deprecation/this-property-fallback",
```

Comment:
This will be switched back to the cos-fork once https://github.com/cos-forks/ember-content-placeholders/pull/6 is merged, which will happen before this is merged.

Reply:
Ok, that sounds good.